app/presenters/nsm/check_answers/application_status_card.rb

```diff
@@ -3,42 +3,124 @@
 module Nsm
   module CheckAnswers
     class ApplicationStatusCard < Base
+      include GovukLinkHelper
+      include GovukVisuallyHiddenHelper
+      include ActionView::Helpers::UrlHelper
       attr_reader :claim
 
+      delegate :status, to: :claim
+
       def initialize(claim)
         @claim = claim
         @group = 'application_status'
         @section = 'application_status'
       end
 
       def row_data
+        response = status == 'submitted' ? nil : '<p>Fake LAA Response</p>'
+        links = join_strings(*edit_links, *appeal_button)
         [
           {
             head_key: 'application_status',
-            text: "<strong class=\"govuk-tag #{I18n.t("nsm.claims.index.status_colour.#{claim.status}")}\">" \
-                  "#{I18n.t("nsm.claims.index.status.#{claim.status}")}</strong>".html_safe
+            text: sanitize(status_text, tags: %w[strong br p])
           },
-          date_actioned_row
+          (if response
+             {
+               head_key: 'laa_response',
+              text: sanitize(response, tags: %w[p]) + sanitize(links, tags: %w[ul li a p])
+             }
+           end)
         ].compact
       end
 
       private
 
-      def date_actioned_row
-        {
-          head_key: date_actioned_head,
-          text: claim.updated_at&.to_fs(:stamp)
-        }
+      def sanitize(value, tags: [])
+        ApplicationController.helpers.sanitize(value, tags:)
       end
 
-      def date_actioned_head
-        case claim.status
-        when 'submitted'
-          'submitted'
-        when 'granted', 'auto_grant', 'part_grant'
-          'assessed'
+      def status_text
+        if status == 'submitted'
+          join_strings(status_tag(status), submitted_data, translate(:awaiting_review), tag.br, claimed_amount)
         else
-          'returned'
+          join_strings(status_tag(status), submitted_data, tag.br, claimed_amount, allowed_amount(status))
+        end
+      end
+
+      def join_strings(*strings)
+        strings.map { |str| str == tag.br ? str : "<p>#{str}</p>" }.join
```

Comment:
This seems really weird to me. If I call `join_strings("foo", "bar", tag.br, "baz")` this will return:

```
<p>foo</p>
<p>bar</p>
<br>
<p>baz</p>
```

But there are already line breaks between bar and baz because they're in separate paragraphs. The br increases the gap between bar and baz but by a very non-standard amount - the design system is very specific about the gaps between paragraphs, and adding extra `br` elements between paragraphs just messes with that.

Reply:
Also, I'm really not a fan of manually building HTML strings that you then have to sanitize. Much better to create sanitized strings in the first place, i.e. replace `"<p>#{str}</p>"` with `tag.p(str)`

Reply:
to the first point... take that up with @mary as that is the design.. :)